fah_data_compressor.py

```diff
@@ -83,11 +82,21 @@ def compress(self, project_root, max_dirs_number):
 
     def run(self):
         """Run the compression."""
-        args = self.get_cli_arguments()
-        project_root = args.projectPath
-        max_dirs_number = args.max_dirs_number
-        self.compress(project_root, max_dirs_number)
+        self.compress()
 
 
 if __name__ == '__main__':
-    FAHDataCompressor().run()
+    ARGUMENT_PARSER = argparse.ArgumentParser()
```

Comment:
Per @xaviermartinez2012 this is a command-line-related item, it shouldn't belong to the FAHDataCompressor class.